VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -232,13 +232,16 @@ private void getRowViewForLeaf(ViewHolder viewHolder,
         viewHolder.rowTitle.setText(unit.getDisplayName());
 
         if (row.component instanceof VideoBlockModel) {
-            final DownloadEntry videoData = ((VideoBlockModel)row.component).getDownloadEntry(storage);
+            final DownloadEntry videoData = ((VideoBlockModel) row.component).getDownloadEntry(storage);
             if (null != videoData) {
                 updateUIForVideo(viewHolder, videoData);
                 return;
             }
         }
-        if (!unit.isMultiDevice()) {
+        if (unit.getType() == BlockType.DISCUSSION) {
```

Comment:
This should also be changed to `unit instanceof DiscussionBlockModel`. Ref https://github.com/edx/edx-app-android/pull/724#discussion_r65496175